compiler-access/src/main/scala/org/alephium/ralph/lsp/access/compiler/parser/soft/ast/Token.scala

```diff
@@ -125,26 +141,16 @@ object Token {
   case class NumberLiteral(lexeme: String) extends Term
   case class StringLiteral(lexeme: String) extends Term
 
-  /**
-   * Order such that tokens such as:
-   *  - `||` come before `|`
-   *  - `+=` come before `+` and `=`
-   */
-  implicit val reservedDescendingOrdering: Ordering[Reserved] = {
-    case (x: Reserved, y: Reserved) =>
-      val lengthCompare =
-        y.lexeme.length compare x.lexeme.length
-
-      if (lengthCompare == 0)
-        y.lexeme compare x.lexeme
-      else
-        lengthCompare
-  }
-
   val reserved: Array[Reserved] =
     EnumerationMacros
       .sealedInstancesOf[Reserved]
       .toArray
       .sorted
 
+  val infix: Array[InfixOperator] =
+    EnumerationMacros
+      .sealedInstancesOf[InfixOperator]
```

Comment:
OMG didn't know we had this macro, I have few places where I could use this in the explorer-backend